src/neuroglancer/datasource/graphene/backend.ts

```diff
@@ -45,6 +45,9 @@ import { DisplayDimensionRenderInfo } from 'neuroglancer/navigation_state';
 import { forEachVisibleSegment } from 'neuroglancer/segmentation_display_state/base';
 import { computeChunkBounds } from 'neuroglancer/sliceview/volume/backend';
 
+import {Uint64OrderedSet} from 'neuroglancer/uint64_ordered_set';
```

Comment:
It seems to me that we don't need the starred segments on the worker thread at all.

If for some reason that is not possible, then the correct way to add it is to add it to config/bundle-config.js in the `chunkWorkerModules` array.

Reply:
I just pushed [3197e91](https://github.com/google/neuroglancer/pull/425/commits/3197e9135c749d3eaa18a07ab185054e2de9d58e)

selectedSegments is in VisibleSegmentsState, it seems tricky to remove. forEachVisibleSegment is used by the chunk workers, at least by graphene, and expects a VisibleSegmentsState.

As long as I don't call `Uint64OrderedSet.makeWithCounterpart`, there is no error though it does mean selectedSegments is undefined if someone tried to access it from a chunk worker.